.circleci/config.yml

```diff
@@ -107,3 +130,17 @@ workflows:
       - screenshotter:
           name: chrome
           browser: chrome
+      - screenshotter:
+          name: safari
+          browser: node
+          browserstack: |
+              {
+                "browserName": "Safari",
+                "browser_version": "13.1",
```

Comment:
Is the mix of snake case and camel case intentional here?

Reply:
@kevinbarabash `browserName` is [an official selenium capability](https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities#used-by-the-selenium-server-for-browser-selection), whereas others are [Browserstack-specific capabilities](https://www.browserstack.com/automate/capabilities).